.goreleaser.yml

```
@@ -0,0 +1,132 @@
+before:
+  hooks:
+    - go version | grep --quiet "go1\.15\.2" || echo "Go binary version must be 1.15.2"
+    - go mod download
+builds:
+  - id: operator-sdk
+    main: ./cmd/operator-sdk
+    binary: operator-sdk
+    env:
+      - CGO_ENABLED=0
+      - GO111MODULE=on
+      - REPO=github.com/operator-framework/operator-sdk
+    mod_timestamp: "{{ .CommitTimestamp }}"
+    asmflags:
+      - all=-trimpath={{ dir .ArtifactPath | dir | dir | dir }}
```

Not sure it matters one way or the other, but another option here is:
```suggestion
      - all=-trimpath={{ dir .Env.PWD }}
```
Might be more intelligible than the repeated `| dir` calls.